config.json

```diff
@@ -259,7 +259,11 @@
     {
       "difficulty": 1,
       "slug": "luhn",
-      "topics": []
+      "topics": [
+	"Algorithms",
+	"Integers",
+	"Strings"
```

Comment:
Yeah I have tabs should be spaces. Mind if I add an [.editorconfig](http://editorconfig.org/) to .gitignore, or perhaps we should have one in the project root for everyone?